webservice/utils/add-user-by-email.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var db = require('../../db')
+  , serializeValue = require('dbjs/_setup/serialize/value')
+  , mano = require('mano')
+  , dbDriver = mano.dbDriver
+  , deferred = require('deferred')
+  , userStorage = dbDriver.getStorage('user')
+  , User = require('../../model/user')(db);
+
+module.exports = function (data) {
+	var def = deferred(), email;
+	try {
+		email = User.prototype.getOwnDescriptor('email').type.validate(data.email);
+		def.resolve(email);
+	} catch (err) {
+		def.reject(err);
+	}
+
+	return def.promise.then(function (email) {
```

Comment:
there s no rejection handler here.
And the promise can be rejected 
https://github.com/egovernment/eregistrations/pull/1912/files#diff-31ab453cd300af474a24755c7e256329R14